landing.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Landing Page</title>  
+    </head>
+    
+
+    <h1>Landing Page</h1>
+    <form>
+        <label for="username">username:</label><br>
+        <input type="text" id="user"><br>
+        <label for="password"> password:</label><br>
+        <input type="text" id="pass"><br>
```

Comment:
The password should be hidden (i.e. when typing passwords into websites, the letters enter in as dots instead of showing the letters)